spec/support/graphiti/schema.json

```diff
@@ -2450,17 +2486,6 @@
             "lt",
             "lte"
           ]
-        },
-        "membership_number": {
-          "type": "integer",
-          "operators": [
-            "eq",
-            "not_eq",
-            "gt",
-            "gte",
-            "lt",
-            "lte"
-          ]
```

Comment:
Why was this removed?

Reply:
there was a typo in [filterable](https://github.com/hitobito/hitobito_sac_cas/pull/711/files#diff-89b0aa447b4c15f373fe22729e9b5456a4333339615c366af5af1e3b760c20e4L13)